Source/Moya.swift

```diff
@@ -240,6 +315,59 @@ public extension MoyaProvider {
 
 internal extension MoyaProvider {
     
+    private func sendUpload(target: Target, request:NSURLRequest, multipartBody:[MultipartFormData], progress: Moya.ProgressBlock? = nil, completion: Moya.Completion) -> CancellableWrapper {
+        var cancellable = CancellableWrapper()
+        let plugins = self.plugins
+        
+        let multipartFormData = { (form:RequestMultipartFormData) -> Void in
+            let params = decodeHTTPBody(request.HTTPBody)
```

Comment:
@hirad I merged your pull request and added the parameter encoding using Alamofire's URL encoding. I don't think we need to support other encoding types on multipart form.